libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/cert/CertRecordStoreConnection.java

```diff
@@ -73,21 +73,13 @@
     int deleteExpiredX509CertRecords(int expiryTimeMins);
 
     /**
-     * Update lastNotifiedServer and lastNotifiedTime for certificate that failed to refresh for more than one day.
+     * List all certificates that failed to refresh for more than one day.
      * @param lastNotifiedServer
      * @param lastNotifiedTime
      * @param provider
      * @return True if at least one certificate record was updated (needs notification to be sent)
```

Comment:
1) by default we were list certs that were not refreshed for more than 3 days (the default value in jdbc) 
2) let's update the return description since we're returning List now.